addons/account/models/chart_template.py

```diff
@@ -998,7 +998,7 @@ def _deref_account_tags(self, template_code, tax_data):
         for tax in tax_data.values():
             for fname in ('invoice_repartition_line_ids', 'refund_repartition_line_ids', 'repartition_line_ids'):
                 if tax.get(fname):
-                    for _command, _id, repartition in tax[fname]:
+                    for repartition in (repartition for (command, _id, repartition) in tax[fname] if command != Command.CLEAR):
```

Comment:
Why not a simple
```python
if command == Command.CLEAR:
    continue
```
instead of that hard to understand comprehension?
Or yet again, some pattern matching?